backend/projects/RepositoryDeployConfig.qml

```diff
@@ -0,0 +1,60 @@
+import QtQuick 2.12
+import "../core"
+
+WSNetworkAbstract {
+    id: root
+
+    property string repositoryName: ""
+
+    com: WSComOne
+    autoReset: true
+
+    request: "sccs:read:/repository/cd/config"
+    dataRequest: {
+        "plugin": Store.sccs_plugin_settings.plugin,
+        "session": Store.sccs_plugin_settings.sessionObj,
+        "repository": repositoryName
+    }
+
+    onDataRequestChanged: repositoryName !== "" && send()
+
+    onDataResponseChanged: {
+
+        if (dataResponse === undefined || dataResponse === null) {
+            return
+        }
+
+        for (const env of dataResponse.environments) {
+            for (var j = 0; j < Store.currentProject.length; j++) {
```

Comment:
to be consistent with the for..of before do you think that a for..of should be a better choice ?